src/clients/api/queries/getPendingRewards/formatOutput/formatToMainPoolPendingRewardGroup.ts

```diff
@@ -1,29 +1,38 @@
+import BigNumber from 'bignumber.js';
 import { ContractCallReturnContext } from 'ethereum-multicall';
 
 import { MainPoolPendingRewardGroup } from '../types';
 import formatRewardSummaryData, { RewardSummary } from './formatRewardSummaryData';
 
 function formatToMainPoolPendingRewardGroup(
   callsReturnContext: ContractCallReturnContext['callsReturnContext'][number],
+  rewardTokenPrices: Record<string, BigNumber>,
 ) {
```

Comment:
Nit: I'd use an object to define these params, so that it can be easily expanded later on if needed.